### PR TITLE
Add HQ email and update user guide URL

### PR DESCRIPTION
**Pull request by avdata99**
Related to #694 

Email at the footer (the client choose the format)

image

User guide link updated

### CODE REVIEW

**Comment by amercader** [comment]
I think I prefer one of these two options:

1. Single box in the footer
Screenshot 2021-09-20 at 13-35-00 Data Containers - RIDL

2. Link in left hand bar
Screenshot 2021-09-20 at 13-38-21 Data Containers - RIDL

I lean towards 2, but check with the client which one they prefer

**Comment by avdata99** [comment]
This PR is ready @amercader

File: ckanext/unhcr/templates/snippets/contact.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="module module-narrow module-shallow">
+  <h2 class="module-heading">
+    <i class="fa fa-book"></i>
```
Comment (amercader) [inline ckanext/unhcr/templates/snippets/contact.html:3]:
```suggestion
    <i class="fa fa-envelope"></i>
```

File: ckanext/unhcr/src/css/footer.css [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
   & .attribution {
     position: absolute;
-    bottom: 15px;
```
Comment (amercader) [inline ckanext/unhcr/src/css/footer.css:14]: Is this needed after moving the link to the sidebar?

**Review by amercader — APPROVED** [review]
Two minor things and it's good to go

File: ckanext/unhcr/src/css/footer.css (hunk 2 above)
Comment (avdata99) [reply ckanext/unhcr/src/css/footer.css:14]: Yes, this is the difference @amercader 

Selection_417

Selection_416

**Pull request merged**